libvast/vast/system/actors.hpp

```diff
@@ -83,6 +83,22 @@ using status_client_actor = typed_actor_fwd<
   caf::replies_to<atom::status, status_verbosity>::with< //
     caf::dictionary<caf::config_value>>>::unwrap;
 
+using store_actor = typed_actor_fwd<
+  // Register an exporter actor.
+  // TODO: This is only for compatibility with the legacy archive and
+  // can be removed with that.
+  caf::reacts_to<atom::exporter, caf::actor>,
+  // Starts handling a query for the given ids.
+  caf::reacts_to<ids, archive_client_actor>,
+  // Erase the events with the given ids.
+  caf::replies_to<atom::erase, ids>::with<atom::done>>::unwrap;
```

Comment:
Should this be a status client actor as well?

Reply:
Not until the old archive is deprecated. Otherwise we would get the same information as part of the status report of every legacy partition.